app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -1,4 +1,6 @@
 React = require 'react'
+strip = require 'strip-markdown'
+remark = (require 'remark').use(strip)
```

Comment:
Sorry, saw this after I'd merged the PR. Another option would be to render markdown in the summary, then style summary images with `display:none`. Or just leave them in.